sdk/messaging/azservicebus/internal/rpc.go

```diff
@@ -0,0 +1,445 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@jhendrixMSFT: this is just a copy of the code from messaging/internal/rpc.go with changes to:
- Remove the retryableRPC function, since retries are handled by callers, not here.
- Fixing a small bug in where we were always returning a detached error. Now we broadcast the actual last error we got, which helps with recovery.

Reply:
Why do we need a copy of the code, is the other one going away?

Reply:
I think GH ate my reply as I don't see it on the main PR page.